_pages/en/it-picture-long-term.md

```diff
@@ -99,11 +112,18 @@ EA teams provide a loosely coupled architecture (reusable infrastructure artifac
 EA is held accountable to:
 
 - Provide an artifact repository which, when used by other teams, puts them into compliance
-- Other teams may freely opt out of their services, and they will be measured against the uptake rate from their clients
+- Produce reference architectures for solution teams to use and align their designs to
+- Recommend investment decision bodies on areas IT needs to invest in
```

Comment:
Recommend? I would argue that's a bit of a fluff word. There's no accountability in 'recommendations' 

What do they do? With what authority?

Reply:
Removed. Until I am able to defend the statement :)
Changed it to be about adherence to compliance (which may require recommending reviewing certain policies and directives that are in conflict)

Reply:
See commit #02a1519 with removal and wordings on compliance